libs/search-widget/src/old-components/row/Row.svelte

```diff
@@ -77,7 +77,7 @@
               on:click={() => searchBy(label)} />
           {/each}
           {#if labels.length > 4}
-            <div class="label">+</div>
+            <Label label={{ labelset: labels[4].labelset, label: '+' }} />
```

Comment:
This `+` label looks suspicious to me. Shouldn't this be a button so you can display the hidden labels?